packages/basic-crawler/src/internals/basic-crawler.ts

```diff
@@ -529,7 +530,8 @@ export class BasicCrawler<Context extends CrawlingContext = BasicCrawlingContext
     protected respectRobotsTxtFile: boolean;
     protected onSkippedRequest?: SkippedRequestCallback;
     private _closeEvents?: boolean;
-
+    private shouldLogMaxProcessedRequestsExceeded = true;
```

Comment:
This doesn't have to be a class field (before it was a closure variable and the usage didn't change)... but the other one has to be a class field, so I guess it's fine to have them both here